### PR TITLE
Remove unnecessary IBM i variants of tests

### PR DESCRIPTION
**Pull request by NattyNarwhal**
These tended to be identical or disabled a working test. For the latter, I suspect IBM i's SQL dialect compatibility caught up.

### CODE REVIEW

File: tests/test_102_NumFieldsSelect_01.phpt [hunk 1]
```diff
@@ -3,7 +3,6 @@ IBM-DB2: db2_num_fields() - select
 --SKIPIF--
 <?php
   require_once('skipif.inc');
-  require_once('skipif3.inc');
```
Comment (kadler) [inline tests/test_102_NumFieldsSelect_01.phpt:6]: This change seems unrelated to all the other changes.

Comment (NattyNarwhal) [reply tests/test_102_NumFieldsSelect_01.phpt:6]: Actually not - it makes the LUW version of this test run on i, since skipif3 handles if a test runs on i (PASE or Db2 Connect). It's incredibly obviously named, no?

I'm guessing like v5r4/6.1 didn't support VALUES...

Comment (kadler) [reply tests/test_102_NumFieldsSelect_01.phpt:6]: Gotcha. From the diff it looks like an outlier since all the other changes were deleting V6 variants of the tests.

Comment (NattyNarwhal) [reply tests/test_102_NumFieldsSelect_01.phpt:6]: Yeah, anything that was TBD removed was prob me restoring the non-PASE version fo the test to work on all platforms...

**Review by kadler — APPROVED** [review]

**Comment by NattyNarwhal** [comment]
I think I'll merge these; if there's more redundant tests, let me know.
